Add detail description of token generation and usage (#106)
ietf-teep/teep-protocol: draft-ietf-teep-protocol.md

  the TEEP Agent.

token
: The value in the token parameter is used to match responses to requests.
  This is particularly useful when a TAM issues multiple concurrent requests
  to a TEEP Agent. The token is not needed when the attestation bit is set in
  the data-item-requested value. The size of the token is at least 8 bytes
  (64 bits) and maximum of 64 bytes, which is the same as in an EAT Nonce

>   Claim. (see {{I-D.ietf-rats-eat}} Section 3.3). The first usage of a token

Reviewer: ```suggestion
  Claim (see {{I-D.ietf-rats-eat}} Section 3.3). The first usage of a token
```

>   generated by TAM MUST be randomly created.

Reviewer: ```suggestion
  generated by a TAM MUST be randomly created.
```

>   Consequent token value MUST be a different value for each request message

Reviewer: ```suggestion
  Consequent token values MUST be different for each request message
```

  the TEEP Agent.

token
: The value in the token parameter is used to match responses to requests.
  This is particularly useful when a TAM issues multiple concurrent requests
  to a TEEP Agent. The token is not needed when the attestation bit is set in
  the data-item-requested value. The size of the token is at least 8 bytes

>   (64 bits) and maximum of 64 bytes, which is the same as Nonce Claim in EAT.

Reviewer: ```suggestion
  (64 bits) and maximum of 64 bytes, which is the same as in an EAT Nonce Claim
```

Author: @dthaler thanks, pushed the change

  (see {{I-D.ietf-rats-eat}} Section 3.3). The first usage of a token
  generated by TAM MUST be randomly created so the device will unlikely
  receive the same token value when the device handles multiple TAMs.
  Consequent token value MUST be a different value for each request message

>   to distinguish the correct response from multiple requests.

Author: @dthaler also pushed the change
